mbbsd/term.c

```diff
@@ -52,16 +52,22 @@ void term_resize(int w, int h)
 
 
     /* make sure reasonable size */
-    h = MAX(24, MIN(100, h));
-    w = MAX(80, MIN(200, w));
+    int h_pref = MAX(24, MIN(100, h));
+    int w_pref = MAX(80, MIN(200, w));
 
-    if (w != t_columns || h != t_lines)
+#ifdef USE_PFTERM
```

Comment:
The `USE_PFTERM` here had a typo `PFTERM` which prevented the fix from resolving the issue of this PR.

This typo has been fixed in the latest force-push.